tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,88 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    Click Link    Resources
+    Sleep    5s
+   ${list_of_tile} =    Get List Of Ids Of Tiles
+   Verify Star Icons Are Clickable    ${list_of_tile}
+
+    FOR  ${index}  IN RANGE    27    48
+         Click Element    //*[@id="${list_of_tile}[${index}]"]/div[1]/span
+         ${clicked} =     Get Element Attribute    //*[@id="${list_of_tile}[${index}]"]/div[1]/span    class
+         Should Be Equal    ${clicked}    odh-dashboard__favorite m-is-favorite
+    END
+
+    ${list_of_new_tile} =    Get List Of Ids Of Tiles
+
+    Lists Should Be Equal    ${list_of_tile}    ${27}    ${48}
+    ...                      ${list_of_new_tile}    ${0}    ${21}
+
+    Click Button    //*[@id="list-view"]
+    Sleep    5s
+
+
+    ${list_of_new_tile} =    Get List Of Atrributes    //div[@class="odh-list-item__doc-title"]    id
+
+    Lists Should Be Equal    ${list_of_new_tile}    ${0}    ${42}
+    ...                      ${list_of_tile}    ${27}    ${48}   ${2}
+
+    Click Button    //*[@id="card-view"]
+    Sleep    2s
+    FOR  ${index}  IN RANGE    27    48
+         Log To Console    ${index}    ${list_of_tile}[${index}]
+         Click Element    //*[@id="${list_of_tile}[${index}]"]/div[1]/span
+         ${clicked} =     Get Element Attribute    //*[@id="${list_of_tile}[${index}]"]/div[1]/span    class
+    END
+
+
```

Comment:
as last thing, I think it would be good to add a teardown to restore the original setting (removing the favorite) and check if everything goes back to the original situation before the user marked the favorite items

Reply:
I have done that in the tc but will move to the teardown

Reply:
When I added it to a teardown, it kept the browser open, shall I add it?

Reply:
Something might be wrong. Please, ping us off-line to try to solve it

Reply:
it happens because if you set the [Teardown] tag in the TC, it replaces the one set as `Test Teardown` in the settings section. So you need to add the kw to close browser in `Remove Items From Favourites`

Reply:
Done